src/cpp/core/r_util/RActiveSessionStorage.cpp

```diff
@@ -112,7 +114,9 @@ Error FileActiveSessionStorage::readProperties(std::map<std::string, std::string
    std::vector<FilePath> files{};
    std::vector<FilePath> failedFiles{};
    pValues->clear();
-   propertyDir.getChildren(files);
+   Error error = propertyDir.getChildren(files);
```

Comment:
nit: Is this change really needed? I think we would already return `Success()` if the propertyDir is not found, since `files` will be empty.

Reply:
Oh yeah, good point. I like the code a little better where we are testing the error just from the readability perspective.

Reply:
You're right, that makes sense to me from a readability perspective.